pyccel/codegen/printing/fcode.py

```diff
@@ -1690,6 +1690,7 @@ def _print_OMP_Parallel_Construct(self, expr):
 
     def _print_Omp_End_Clause(self, expr):
         omp_expr = str(expr.txt)
+        omp_expr = omp_expr.replace("for", "do")
```

Comment:
Can `omp_expr` contain any user defined words? E.g. the names of variables? If so what happens if the variable is called "my_for_var"?

Reply:
No, we can't have variables in the `END Clause`, variables must be declared at the start of the` loop construct `